recipes/marisa-trie/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{%set name = "marisa-trie" %}
```

Comment:
The problem with that strategy is it results in a loss of functionality in `marisa-trie` when that changes to not include Python. This would be surprising to many users and would arguably be a bug.